_includes/header.html

```diff
@@ -8,7 +8,12 @@
       <!--Import Google Icon Font-->
       <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
       <!--Import materialize.css-->
-      <link type="text/css" rel="stylesheet" href="{{site.baseurl}}/assets/materialize/css/materialize.min.css"  media="screen,projection"/>
+      {% if jekyll.environment == "development" %}
```

Comment:
Hmm, are you sure this is the correct way to do this? :thinking: See deploy preview at netlify, it does not load the CDN.

Reply:
We may need to change the build command for production, but I don't see it anywhere in the code base where I can update it.

Here's the Jekyll docs for determining the environment:
https://jekyllrb.com/docs/configuration/environments/

It mentions to export the environment variable:
```bash
JEKYLL_ENV=production jekyll build
```

Anything I can do to update the build command?

Reply:
Hmm, I guess I need to change the build command at netlify. I will merge it.

Reply:
Checked the site after merge, it's calling the CDN links. 😄 
And yup, I think updating the netlify build command should make it work for deploy preview.

Thanks for merging @ayan-b!